site.yml

```diff
@@ -13,6 +13,14 @@
       with_items: "{{ groups['all'] }}"
       when: inventory_hostname == play_hosts[0]
 
+- name: preflight checks
+  hosts: all:!vyatta-*
+  roles:
+    - role: preflight-checks
+      run_once: true
```

Comment:
Why is this run_once set for the entire thing? Seems wrong. Let the tasks within decide if they should run once.